_posts/2022-11-30-extend-clang-to-resugar-template-specialization-accesses.md

```diff
@@ -120,7 +118,7 @@ increase my confidence as a developer in the LLVM open community!
 
 **Contact me!**
 
-Email: mizvekov@gmail.com
+Email: [mizvekov@gmail.com](/team/MatheusIzvekov)
```

Comment:
I meant that we should use the email as is without extra punctuation. In the `blog-post-card.html` or where the rendering happens we can turn it into a link.

Reply:
Eg you want to move the email to the page header?  This seems like it is a free text field currently

Reply:
No, I do not want to have markdown style links or any other decoration around the email because that field is used (and will be used) as a unique ID for the people. Based on that we can make more advanced queries in the text files. That is why we should not decorate with more. Instead, we could move the decoration where it is needed - that is the blog post visualization...

Reply:
eg, something is grepping the markdown for the word Email?

Reply:
No, something makes a selection based on the email and/or author name across yml files. Or something could if we have a unique id. Pretty much how any relational database works.

Reply:
that makes sense.. but this is just some free text. So I don't understand the point.

Reply:
but you need to parse free text for that (so I guess the comment above is incorrect)... 

Should not use the header information for that? (which does not make the life of the interested person viewing the page more difficult and could be used to make a contact form standard across posts..)

well it seems the correct action is to remove the part of the blog that is being discussed. The contact information is already at the top of the post. No need to repeat it.

Reply:
Wait, dammit.... I thought my comment was under the `yaml` file. Please ignore, now I can understand why what I was saying does not make any sense...